editor/hooks/copy-content/index.js

```diff
@@ -44,8 +36,16 @@ class CopyContentButton extends Component {
 	}
 }
 
-export default connect(
-	( state ) => ( {
-		editedPostContent: getEditedPostContent( state ),
-	} )
-)( CopyContentButton );
+const Enhanced = query( ( select ) => ( {
```

Comment:
We have also `withState` HOC created by @aduth. I'm wondering if it would fit here :)

Reply:
Agreed, done.